network-api/networkapi/wagtailpages/rss.py

```diff
@@ -16,7 +18,16 @@ class RSSFeed(Feed):
     feed_url = 'https://foundation.mozilla.org/blog/rss/'
     description = 'The Mozilla Foundation Blog'
 
+    def __call__(self, request, *args, **kwargs):
+        # get locale from request header, or fall back to the default EN locale.
+        request_locale = get_locale_from_request(request)
```

Comment:
get_locale_from_request will either return the users locale, or will default to the english one. 

So we are setting the locale here, and then grabbing it in the "items" function, with no need to set default there.